_posts/2021-03-04-PC-Engines-Firmware-v4-13-0-5.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.13.0.5"
+date:   2021-04-02
+categories: Firmware
+---
+
+# PC Engines apu coreboot Open Source Firmware v4.13.0.5
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** e7a68ec.
+2. **Added** resource allocator v4 support. It supports multiple ranges for
```

Comment:
PR is not merged https://github.com/pcengines/coreboot/pull/465
Please remove

Reply:
Removed.